vm/src/cformat.rs

```diff
@@ -218,15 +218,39 @@ fn try_update_quantity_from_element(
     }
 }
 
+fn try_update_adjust_from_tuple(
+    vm: &VirtualMachine,
+    element: Option<&PyObjectRef>,
+    f: &mut CConversionFlags,
+) -> PyResult<()> {
```

Comment:
```suggestion
) -> PyResult<CConversionFlags> {
```
I prefer to have immutable parameters when possible

Reply:
That is fair. I think I was trying to mimic what other functions nearby were doing, but that is a fair suggestion.